index.html

```diff
@@ -195,6 +243,7 @@ <h2 class="contact-me ">Contact me</h2>
             <button class="form-item form-button" type="submit">Get in touch</button>
         </form>
         <img class="home-shape" src="images/Shape.svg" alt="home Shape">
+    </div>
```

Comment:
- [ ] As requested by the previous reviewer you desktop version should be similar to the Figma design as please use the media query for the desktop version. Also please adjust the spacing of you design. You can compare the design below and make the necessary changes.
### Your design
![his](https://github.com/ausiemass/portfolio-austin/assets/81727988/27553fec-0e0d-4a7c-b5ae-dcb5dd4b551a)

### Figma design
![fimga](https://github.com/ausiemass/portfolio-austin/assets/81727988/84867c44-89ce-4c56-85ae-30df637ca268)